tests/exporters/test_vhi.py

```diff
@@ -0,0 +1,68 @@
+from pathlib import Path
+# from unittest.mock import patch, Mock
+import pytest
+
+from src.exporters.vhi import (
+    VHIExporter,
+    _parse_time_from_filename,
+    make_filename,
+    # download_file_from_ftp
+)
+
+import sys
+# get the root project directory
+project_dir = Path(__file__).resolve().parents[2]
+# append to sys
+sys.path.append(project_dir.as_posix())
+
+
+class TestVHIExporter:
+
+    def test_initialisation(self):
+        VHIExporter()
+        return
+
+    def test_filename_illegitimate(self):
+        with pytest.raises(Exception):
+            make_filename(
+                raw_folder=project_dir / "data",
+                raw_filename='this/should/fail.nc',
+                dataset='vhi',
+            )
+
+        return
```

Comment:
Same as above; empty returns are unnecessary

Reply:
think i've fixed all of these